projects/packages/jetpack-mu-wpcom/src/features/wpcom-global-styles/index.php

```diff
@@ -337,6 +350,15 @@ function wpcom_global_styles_in_use_by_wp_global_styles_post( array $wp_global_s
 	// Some keys are ignored because they are not relevant to a custom style
 	// behaviours are not relevant if blank - as they where when included during GB16.4 and later removed.
 	$ignored_keys = array( 'version', 'isGlobalStylesUserThemeJSON' );
+
+	if ( wpcom_site_has_feature( WPCOM_Features::CUSTOM_DESIGN ) ) {
+		unset( $global_styles_content['styles']['css'] );
+
+		if ( array_key_exists( 'styles', $global_styles_content ) && count( $global_styles_content['styles'] ) === 0 ) {
```

Comment:
The condition be reduced to: `if ( empty( $global_styles_content['styles'] ) ) {`